src/json-rpc-api.ts

```diff
@@ -6,4 +6,6 @@ import { handleSubscriptionsClose } from './json-rpc-handlers/subscription/index
 export const jsonRpcApi = new JsonRpcRouter();
 
 jsonRpcApi.on('dwn.processMessage', handleDwnProcessMessage);
-jsonRpcApi.on('subscription.close', handleSubscriptionsClose);
+jsonRpcApi.on('rpc.subscribe.dwn.processMessage', handleDwnProcessMessage);
```

Comment:
Any reason for this particular name?

Consider names with symmetry, perhaps

`rpc.subscribe.open/close` or `dwn.subscription.subscribe/close` or `rpc.subscription.open/close` etc.

Reply:
The idea here was to keep the `rpc.subscription` prefix as a part of a more general purpose subscribe paradigm, so it would be calling `rpc.subscription.{JsonRpcMethodName}`.

I'd still like to keep it that way, but It's not a hill I'd die on.

Reply:
Discussed offline, can revisit if needed.